.github/workflows/main.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - run: pytest *
```

Comment:
I tried  pytest locally, and had to change this to `*.py`, rather than just `*`.  Does this work for you?

Reply:
Yes, you are right.